Leanplum-iOS-SDK-source.podspec

```diff
@@ -26,6 +26,7 @@ Analytics.
   s.frameworks = 'CFNetwork', 'Foundation', 'Security', 'SystemConfiguration', 'UIKit'
   s.weak_frameworks = 'AdSupport', 'StoreKit'
   s.library = 'sqlite3'
+  s.dependency 'Mantle'
```

Comment:
1) This will be the first hard dependency with 3rd party SDK. Make sure to talk to @afirst about this.
2) If you are going to add Mantle, it should be used everywhere and not for this one feature, in a different commit.

IMO I'm not against using Mantle, but I am against using it for this one particular place. We can reduce a lot of code by using it on every Requests.

Also we should consider other JSON serializer.

Reply:
1. I made Andrew aware on Slack but will double-check with him.
2. The plan is to move all the requests/responses to a serializer/deserializer going forward. 

I spoke with Lev and we decided that moving to it incrementally was the way to go. Hence, since this is a new model, we will add it here, and migrate the rest over slowly. If you don't think this is the right way, let me know. It is a chicken-and-egg problem. Yes, everything should use a serializer AND everything should be consistent. However, we are not using a serializer currently. What is the incremental way to move towards the ideal world?

I did a quick survey of JSON serializers but am open to other suggestions, definitely not married to Mantle